spec/e2e/support/target-blank.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef SimplifiedLink
+ *
+ * @prop {string=} innerText
+ * @prop {string=} href
+ * @prop {string=} className
+ * @prop {string=} target
+ */
+
+/** @type {import('puppeteer').Page} */
+/** @return {Promise<SimplifiedLink[]>} */
+async function getLinks(page) {
+  return await page.evaluate(() =>
+    Array.prototype.map.call(document.querySelectorAll('a'), (a) => {
+      // There's a weird serialization boundary here so we encode the links as a structure
+      return {
+        innerText: a.innerText.trim(),
+        href: a.href,
+        className: a.className,
+        target: a.target,
+      };
+    }),
+  );
+}
+
+/**
+ * @param {SimplifiedLink} a
+ */
+function toNotHaveTargetBlank(a) {
+  return {
+    pass: a.target != '_blank',
```

Comment:
I haven't gotten around to setting `eslint:recommended` or `airbnb` as the base ESLint configuration, but in both cases we'll eventually encounter this rule:

https://eslint.org/docs/rules/eqeqeq

i.e.

```suggestion
    pass: a.target !== '_blank',
```